packages/server/utils/analytics/analytics.ts

```diff
@@ -306,6 +308,10 @@ class Analytics {
     this.track(userId, 'Task Estimate Set', taskEstimateProperties)
   }
 
+  toggleSubToSummaryEmail = (userId: string, subscribeToSummaryEmail: boolean) => {
+    this.track(userId, 'Summary Email Setting Changed', subscribeToSummaryEmail)
```

Comment:
The 3rd argument of `this.track` expects a dictionary so this should be `this.track(userId, 'Summary Email Setting Changed', {subscribeToSummaryEmail})`